config/routes.rb

```diff
@@ -5,6 +5,12 @@
   # root "articles#index"
   root "users#index"
 
+
+  get 'posts/new', to: 'posts#new'
+  post 'posts', to: 'posts#create'
+  post 'likes/:post_id', to: 'likes#create'
+  post 'comments/:post_id', to: 'comments#create'
+
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
   end
```

Comment:
- [ ] The comments, posts, and likes routes should be wrapped within the `posts` so that we can access the `post_id`. We can nest the first four routes within the users and posts routes resources.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show, :new, :create] do
      resources :comments, only: [:create]
      resources :likes, only: [:create]
    end
  end
  ```